### ISSUE TITLE
Bug: "Show Menu" not in tab order

### ISSUE THREAD

**Issue by kcarrandale** [ProgressiveCoders/Spoke#1928]
**Describe the bug**
When navigating the admin section of Spoke using only the keyboard, the "Show Menu" button cannot be tabbed to, so the menu is stuck closed unless you refresh.

**To Reproduce**
1. Go to /admin/1/campaigns
2. Use the tab key to go to the "Close Menu" button and press enter.
3. Use tab/shift-tab to try to navigate to the "Show Menu" button. 
4. Tabbing skips from the User Icon Menu to the page's main content.

**Expected behavior**
You should be able to tab to the "Show Menu" button and activate it with either space or enter.

**Screenshots**
N/A

**Platform (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Desktop
 - Version 10.0

**Additional context**
The other main issue I've found so far is that aside from the "Open Menu" button (which is an actual button), none of the "buttons" (divs?) activate when pressing space or enter. But I figured that made sense to add as a separate bug.

I'm new to this, so I'm not sure what's practical within React/Spoke, but possible fixes for this type of issue include:
- Making "Show Menu" a button element (like "Close Menu" is) to use buttons' default tabbing and activation behavior.
- Adding a tabindex of zero to the "Close Menu" div when it is visible. (This fixes the tabbing, but not the inability to activate it with the keyboard.)

More info: [WebAIM: Keyboard Accessibility](https://webaim.org/techniques/keyboard/)

**Comment by oburbank** [comment]
I have a fix incoming for this. It seems like it is difficult to have vertical text w/in a button, so a work around was found by creating a clickable span instead, unfortunately span's are not focusable by default. I'm replacing "SHOW MENU" and "CLOSE MENU" with icons and `IconButton`s, negating the need for creating a clickable element with vertical text

**Comment by kcarrandale** [comment]
Very belated - cool! Do you know if the `IconButton`s are understandable to screen reader users? I'm not very familiar with React, but accessible icon buttons often involve something like screenreader-only text. [Accessible Buttons and Icon Links](https://www.smashingmagazine.com/2021/03/complete-guide-accessible-front-end-components/#accessible-buttons-and-icon-links)
